src/sonic-ctrmgrd/tests/ctrmgr_iptables_test.py

```diff
@@ -103,30 +103,29 @@ def __init__(self, ret, stdout, stderr):
 
 
 def mock_subproc_run(cmd, shell, capture_output):
-    cmd_prefix = "sudo iptables -t nat "
-    list_cmd = "{}-n -L OUTPUT ".format(cmd_prefix)
-    del_cmd = "{}-D OUTPUT ".format(cmd_prefix)
-    ins_cmd = "{}-A OUTPUT -p tcp -d ".format(cmd_prefix)
+    list_cmd = ["sudo", "iptables", "-t", "nat", "-n", "-L", "OUTPUT"]
+    del_cmd = ["sudo", "iptables", "-t", "nat", "-D", "OUTPUT"]
+    ins_cmd = ["sudo", "iptables", "-t", "nat", "-A", "OUTPUT", "-p", "tcp", "-d"]
 
-    assert shell
+    assert shell == False
```

Comment:
>shell == False [](http://example.com/codeflow?start=11&length=14)

Why change it?

Reply:
Because subprocess.run should be used with shell=False instead of True in original code. https://github.com/sonic-net/sonic-buildimage/blob/8d7f66ab175aaca05395314a26b2b1dab85eb576/src/sonic-ctrmgrd/ctrmgr/ctrmgr_iptables.py#L109